### PR TITLE
CORE-5247 Fixes for issues related to running combined worker

### PR DESCRIPTION
**Pull request by mbrkic-r3**
Fixes for issues related to running combined worker:

- Fixed boot DB parameters
- Made property "schema.name" local

### CODE REVIEW

File: data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml [hunk 1]
```diff
@@ -2,8 +2,15 @@
...
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+    <property name="schema.name" value="CRYPTO" global="false"/>
+
     <changeSet author="R3.Corda" id="crypto-creation-v1.0">
-        <createTable tableName="crypto_wrapping_key">
+
+        <sql>
+            CREATE SCHEMA IF NOT EXISTS CRYPTO;
+        </sql>
```
Comment (driessamyn) [inline data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml]: Ideally this shouldn't be in here as the schema should be created outside of this. I thought the cluster bootstrapper already did that?
We can leave this in, it'll just mean we have to take it out later on.

File: data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml [hunk 2]
```diff
@@ -22,9 +29,10 @@
         </createTable>
         <addPrimaryKey  columnNames="alias"
                         tableName="crypto_wrapping_key"
-                        constraintName="crypto_wrapping_key_pk"/>
+                        constraintName="crypto_wrapping_key_pk"
+                        schemaName="${schema.name}"/>
 
-        <createTable tableName="crypto_hsm_config">
+        <createTable tableName="crypto_hsm_config" schemaName="${schema.name}">
```
Comment (driessamyn) [inline data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml]: Similar to the above, schemas were left out as they should be passed into the connection string to this DB.

File: data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml (hunk 1 above)
Comment (alexey-kadyrov-r3) [reply data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml]: it has to be deleted

Comment (alexey-kadyrov-r3) [reply data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml]: That script is only part of the migration, there is another script which runs for the cluster and for a vnode (vnode-crypto directory).

**Review by driessamyn — DISMISSED** [review]

**Review by driessamyn — APPROVED** [review]
LGTM

**Review by driessamyn — APPROVED** [review]
LGTM

**Pull request merged**